plenum/test/view_change/test_future_vc_done.py

```diff
@@ -0,0 +1,75 @@
+from plenum.common.messages.node_messages import ViewChangeDone, FutureViewChangeDone
+
+
+def test_future_vcdone_with_is_initial_propagate_primary_no_quorum(fake_node):
+    """
+    Check, that view_change would be not started without quorum of propagate_primary view_change_done messages
+    if is_initial_propagate_primary is True
+    """
+    frm = 'Node3'
+    current_view = fake_node.view_changer.last_completed_view_no
+    proposed_view_no = current_view + 1
+    msg = ViewChangeDone(proposed_view_no, frm, fake_node.ledger_summary)
+    fake_node.view_changer._next_view_indications[proposed_view_no] = {}
+    fake_node.view_changer._next_view_indications[proposed_view_no][frm] = msg
+    view_changer = fake_node.view_changer
+    res = view_changer._start_view_change_if_possible(proposed_view_no,
+                                                      is_initial_propagate_primary=True)
+    assert res is False
+
+
+def test_future_vcdone_with_is_initial_propagate_primary_with_quorum(fake_node):
+    """
+    Check, that view_change would be started with quorum of propagate_primary view_change_done messages
+    if is_initial_propagate_primary is True
+    """
+    quorum = fake_node.quorums.propagate_primary.value
```

Comment:
It's better to specify the quorum explicitly in the test (to be sure that this is f+1, not occasionally changed to n-f).